docs/guides/Quick Start/03-orders.md

```diff
@@ -0,0 +1,182 @@
+
+### Order
+
+On creating a new order, the order state is set to *open* by default. Order state can be transitioned among `cancel`, `complete`, `reopen`, and `process`. 
+
+Below is a sample request to change the state of order to *complete*:
+
+```json http
+{
+  "method": "post",
+  "url": "http://localhost/api/_action/order/558efc15fe604829b4d0607df75187e0/state/complete",
+  "headers": {
+    "Content-Type": "application/json",
+    "Accept": "application/json",
+    "Authorization": "Bearer Your_API_Key"
+  },
+  "body": {
+  }
+```
+A *canceled* order cannot change to an *in-progress* state unless it is reopened again.
+
+### Order delivery
+
+The order delivery state represents the state of the delivery. `reopen`, `ship`, `ship_partially`, `cancel`, `retour`, and `retour_partially` are the states associated with order delivery.
+
+Below is a sample request that sets the delivery state to *fail*:
+
+```json http
+{
+  "method": "post",
+  "url": "http://localhost/api/_action/order_delivery/558efc15fe604829b4d0607df75187e0/state/fail",
+  "headers": {
+    "Content-Type": "application/json",
+    "Accept": "application/json",
+    "Authorization": "Bearer Your_API_Key"
+  }
+}
+```
+
+### Order transaction
+
+The order transaction state represents the state of the payment. `open`, `fail`, `authorize`, `refund_partially`, `refund`, `do_pay`, `paid`, `paid_partially`, `remind`,  and `cancel` are the states associated with order transaction.
+
+Below is a sample request that sets the payment state to *open*:
+
+```json http
+{
+  "method": "post",
+  "url": "http://localhost/api/_action/order_transaction/558efc15fe604829b4d0607df75187e0/state/open",
+  "headers": {
+    "Content-Type": "application/json",
+    "Accept": "application/json",
+    "Authorization": "Bearer Your_API_Key"
+  }
+  }
+```  
+
+## Refund Payment
+
+[Initiating and capturing payment is handled by store-api](https://shopware.stoplight.io/docs/store-api/8218801e50fe5-handling-the-payment), whereas the admin API deals with refund payment.
+
+The refund payment method can be called only for transactions that are claimed to be successful. 
+
+Generally, refunds are linked to a specific order transaction capture ID. An order can have one or more line items and amounts. Each of these line items signifies refund positions. Based on the number of line items requested for refund, the payment can either be partially or wholly refunded.
+
+To allow easy refund handling, have your payment handler implement the `RefundPaymentHandlerInterface`. Your RefundHandler implementation will be called with the `refundId` to call your PSP or similar.
+
+Use the Refund API endpoint `/api/_action/order_transaction_capture_refund/{refundId}`  to refund payments. You usually need the `refundId` and `context`. The refundId is the id of the `OrderTransactionCaptureRefund` entity, which the payment plugin has created before and will be forwarded to your RefundHandler implementation.
```

Comment:
This requires in-depth knowledge about Shopware Extension (Plugin) Development, so we should mark it somehow (maybe with a warning) and redirect to resources for extension developers. 

```suggestion
<!-- theme: danger -->

> **Refund handling internals** 
>
> To allow easy refund handling, have your payment handler implement the `RefundPaymentHandlerInterface`. Your refund handler implementation will be called with the `refundId` to call the PSP or gateway.
>
> For more information check our documentation on [implementing payment refund handlers](https://developer.shopware.com/docs/guides/plugins/plugins/checkout/payment/add-payment-plugin#refund-example)
```

![image](https://user-images.githubusercontent.com/7780750/203100234-3dad675d-df6c-4b06-865a-8f852cabe624.png)